cmd/main.go

```diff
@@ -17,16 +17,18 @@ import (
 	oplog "github.com/ethereum-optimism/optimism/op-service/log"
 
 	"github.com/ethereum/go-ethereum/log"
-	"github.com/ethereum/go-ethereum/params"
 
 	"github.com/joho/godotenv"
 	"github.com/urfave/cli/v2"
 )
 
 var (
-	GitCommit = ""
-	GitDate   = ""
+	version = "dev"
+	commit  = "none"
+	date    = "unknown"
```

Comment:
are these being used for anything?

Reply:
yep just updated so they're included in the string version